exchanges/poloniex/currency_details.go

```diff
@@ -0,0 +1,235 @@
+package poloniex
+
+import (
+	"errors"
+	"strconv"
+	"sync"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+// CurrencyDetails stores a map of currencies associated with their ID
+type CurrencyDetails struct {
+	pairs map[float64]PairSummaryInfo
+	codes map[float64]CodeSummaryInfo
+	// Mutex used for future when we will periodically update this table every
+	// week or so in production
```

Comment:
Its already being used? Do you mean like "todo: routinely check currency details for long running application"?

Reply:
"todo: routinely check currency details for long running application"